src/core/MOM_continuity_PPM.F90

```diff
@@ -2750,7 +2773,9 @@ subroutine continuity_PPM_init(Time, G, GV, US, param_file, diag, CS)
                  "If true, use the marginal face areas from the continuity "//&
                  "solver for use as the weights in the barotropic solver. "//&
                  "Otherwise use the transport averaged areas.", default=.true.)
-
+  call get_param(param_file, mdl, "VISC_REM_CONT_HVEL_FIX", CS%visc_rem_hvel_fix, &
+                 "If true, velocity cell thickness h_[uv] from the continuity solver "//&
+                 "is not multiplied by visc_rem_[uv].", default=.false.)
   CS%diag => diag
```

Comment:
Please add a comment here suggesting that the default for VISC_REM_CONT_HVEL_FIX should be set to true eventually, and perhaps also add this to the shared MOM6 google drive spreadsheet where we coordinate changes to defaults.